test/integration/apiserver/flowcontrol/concurrency_test.go

```diff
@@ -104,22 +105,28 @@ func TestPriorityLevelIsolation(t *testing.T) {
 	}
 
 	stopCh := make(chan struct{})
-	defer close(stopCh)
+	wg := sync.WaitGroup{}
```

Comment:
nit: traditionally we declare WaitGroup as `var wg sync.WaitGroup`

Reply:
I wouldn't say there is convention in our codebase:
```
wojtekt@wojtekt-l:~/go/src/k8s.io/kubernetes$ git grep "var wg" | wc -l
106
wojtekt@wojtekt-l:~/go/src/k8s.io/kubernetes$ git grep "wg := " | wc -l
82
```